wagtail/admin/forms/tags.py

```diff
@@ -31,8 +33,21 @@ def __init__(self, *args, **kwargs):
             self.widget.free_tagging = self.free_tagging
 
     def clean(self, value):
+
         value = super().clean(value)
 
+        max_tag_length = self.tag_model.name.field.max_length
+        value_too_long = ""
+        for val in value:
+            if len(val) > max_tag_length:
+                if value_too_long:
+                    value_too_long += ", "
+                value_too_long += val
+        if value_too_long:
+            raise ValidationError(
+                _(f"Tag(s) {value_too_long} are over {max_tag_length} characters")
```

Comment:
there is an issue with using f-strings and translations with Transifex. Reference: #6238

please use % formatting. something like:

```suggestion
                _("Tag(s) %{value_too_long}s are over %{max_tag_length}d characters" % {
                    "value_too_long": value_too_long,
                    "max_tag_length": max_tag_length
                })
```

Reply:
@zerolab fixed. thank you for pointing that out!